host-configs/lc-builds/hip.cmake

```diff
@@ -29,17 +29,24 @@ if (ENABLE_OPENMP)
 	set(HIP_COMMON_OPT_FLAGS "-fopenmp ${HIP_COMMON_OPT_FLAGS}")
 endif()
 
+#if(CMAKE_BUILD_TYPE MATCHES Release)
+#  set(RAJA_HIPCC_FLAGS -O2; ${HIP_COMMON_OPT_FLAGS}; ${HOST_OPT_FLAGS} CACHE LIST "")
+#elseif(CMAKE_BUILD_TYPE MATCHES RelWithDebInfo)
+#  set(RAJA_HIPCC_FLAGS -g; -G; -O2; ${HIP_COMMON_OPT_FLAGS}; ${HOST_OPT_FLAGS} CACHE LIST "")
+#elseif(CMAKE_BUILD_TYPE MATCHES Debug)
+#  set(RAJA_HIPCC_FLAGS -g; -G; -O0; ${HIP_COMMON_DEBUG_FLAGS}; CACHE LIST "")
+#endif()
```

Comment:
Should this be left here?

Reply:
Good catch. I forgot to clean that up.  Thanks.

Reply:
Unfortunately, this went into develop yesterday. I just put up a PR to address that. After that is merged, that will propagate to this branch.